.buildkite/pipeline.yml

```diff
@@ -0,0 +1,44 @@
+env:
+  SEGMENT_CONTEXTS: "snyk,npm,aws-credentials,ecr,saucelabs"
```

Comment:
I think we are going to need `npm-publish` (I think that's the name). It's a context with write access on NPM.

Reply:
@CarlosMecha I talked with tooling and updated the npm context to `npm-publish` so it has read and write access, as well as pass the `NPM_TOKEN` env var to the plugin as well in the publish step.